package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "clever",
-  "version": "0.10.0",
+  "version": "0.11.0",
```

Comment:
Is this a backwards incompatible change? Seems like it's either a minor or a patch (both which are represented by bumping the patch version since we're < 1.0.0).